sources/packages/forms/src/form-definitions/sfaa2025-26.json

```diff
@@ -8771,9 +8770,10 @@
               "hideLabel": true,
               "tags": [],
               "conditional": {
-                "show": "true",
-                "when": "youthInCare",
-                "eq": "yes"
+                "show": "",
+                "when": null,
+                "eq": "",
+                "json": ""
```

Comment:
Thanks for the change. Please create a ticket to fix the same in other PY forms.